compiler/rustc_resolve/src/access_levels.rs

```diff
@@ -64,12 +63,11 @@ impl<'r, 'a> AccessLevelsVisitor<'r, 'a> {
                         self.update(
                             id,
                             binding.vis.expect_local(),
-                            prev_parent_id,
+                            module_id,
```

Comment:
I don't think this change is correct.
Eff visibility of an import is indeed limited by the previous import in the chain, not the original module.